contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -30,7 +31,7 @@ import (
 func TestUnary(t *testing.T) {
 	assert := assert.New(t)
 
-	rig, err := newRig(true)
+	rig, err := newRig(true, WithRequestTags())
```

Comment:
Can we add some assertions to another test to make sure that request and metadata tags are *not* set when these options are missing. We could use `TestPass` or `TestChild` to sneak it in.

Reply:
Oh, good ideas. I'll work on it.

Reply:
Done!